subtests/docker_cli/systemd/p4321.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Systemd for p4321 service
```

Comment:
It might be helpful to note that this is a scratch/test service, and perhaps include a note of the form "this is part of docker-autotest, if you see it on a live system it means something has failed very badly". The purpose is to realize that there's a small window during which a test crash could leave this file in the system, and someone might find it, and a helpful comment will save them from wasting time.